Steps4Impact/Journey/Cells/JourneyDetailCell.swift

```diff
@@ -0,0 +1,156 @@
+  }()
+  
+  override func commonInit() {
+    super.commonInit()
+    
+    contentView.addSubview(titleLabel) {
+      $0.top.equalToSuperview().offset(Style.Padding.p20)
+      $0.leading.equalToSuperview().offset(Style.Padding.p24)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p24)
+    }
+    
+    contentView.addSubview(subtitleLabel) {
+      $0.top.equalTo(titleLabel.snp.bottom).offset(Style.Padding.p8)
+      $0.leading.equalToSuperview().offset(Style.Padding.p24)
+      $0.trailing.equalToSuperview().inset(Style.Padding.p24)
+    }
+    
+    contentView.addSubview(bodyText) {
+      $0.top.equalTo(subtitleLabel.snp.bottom).offset(Style.Padding.p16)
+      $0.leading.equalToSuperview().offset(Style.Padding.p24)
+      $0.trailing.bottom.equalToSuperview().inset(Style.Padding.p24)
+      $0.bottom.equalToSuperview()
+    }
+    
+    titleLabel.textColor = Style.Colors.FoundationGrey
+    subtitleLabel.textColor = Style.Colors.FoundationGrey
+  }
+  
+  func configure(context: CellContext) {
+    guard let detailContext = context as? JourneyDetailContext,
+      let milestone = detailContext.milestone else { return }
+    
+    titleLabel.text = milestone.name
+    subtitleLabel.text = milestone.subtitle
+    bodyText.attributedText = getContentAttributedText(from: milestone)
+  }
+}
+
+extension JourneyDetailCell {
+  func getContentAttributedText(from milestone: Milestone) -> NSMutableAttributedString {
+    let htmlstring = milestone.content
+    let stringtext = htmlstring.html2String
+    let result = NSMutableAttributedString()
+    let split = stringtext.components(separatedBy: "https")
+    
+    if split.count > 1 {
+      let stringURL = "https\(String(split[1]))"
+      let attributedString = NSMutableAttributedString(string: stringURL)
+      if let url = URL(string: stringURL) {
+        // Set the 'click here' substring to be the link
+        attributedString.setAttributes([.link: url], range: NSMakeRange(0, stringURL.count))
+      }
+      
+      // Set how links should appear: blue and underlined
+      self.bodyText.linkTextAttributes = [
+        .foregroundColor: UIColor.blue,
+        .underlineStyle: NSUnderlineStyle.single.rawValue
+      ]
+      delegate?.didGetShareText("\(milestone.subtitle): \(stringURL)")
+      let partone = NSMutableAttributedString(string: String(split[0]))
+      let parttwo = NSMutableAttributedString(attributedString: attributedString)
+      result.append(partone)
+      result.append(parttwo)
+    } else {
+      delegate?.didGetShareText(milestone.subtitle)
+      result.append(NSMutableAttributedString(string: stringtext))
+    }
+    
+    let paragraphStyle = NSMutableParagraphStyle()
+    paragraphStyle.lineSpacing = Style.Size.s8
+    result.addAttribute(.paragraphStyle, value: paragraphStyle, range: NSMakeRange(0, result.length))
+    result.addAttribute(.font, value: Style.Typography.smallRegular.font as Any, range: NSMakeRange(0, result.length))
```

Comment:
Not a fan of the `as Any` for attributed text but there really isn't another way without unwrapping it or changing the style struct to not return optionals

Reply:
I can add `if let` before that line to get rid of `as Any`. What do you say?

Reply:
That would work :)

Reply:
Done